presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorColumnHandle.java

```diff
@@ -114,6 +118,11 @@ public boolean isShardUuid()
         return isShardUuidColumn(columnId);
     }
 
+    public boolean isBucketNumber()
```

Comment:
Right now these are both used in shard predicate. That can be rewritten using only the column id.